src/components/KTable/ColumnVisibilityMenu.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="k-table-column-visibility-menu">
+    <KDropdown>
+      <KTooltip text="Show/Hide Columns">
+        <KButton
+          appearance="secondary"
+          class="menu-button"
+        >
+          <template #icon>
+            <CogIcon />
```

Comment:
question: did we decide it will the cog icon? I remember we were discussing using columns icon or something

Reply:
Yes I think a different icon would be more ideal. Material Symbols has some column icons I believe we could add to `@kong/icons`

Reply:
Picking up the new icon can be addressed in a followup PR once we have the desired icon in `@kong/icon`

Reply:
TBH I don't think this is ideal; if the table starts getting used before then, it's going to be confusing for Konnect users to see a random "settings" icon and then have it change a few days later. 

You should be able to get an icon pretty easily from Jason in `#ask-kong-design-system`

Reply:
^ Second that, let's use the proper icon from beginning - I don't think design will have strong opinions against the columns icon